### PR TITLE
Account design new

### LINKED ISSUE #10: Load images from server
Load images from server using base URL. Right now we are not loading any image from a server.

### LINKED ISSUE #11: security at project compiling
@gyosida, @cpinan 

What do you think of improving our server base url protection? We can put the base on gradle file and pull them from a local.properties?

Although our endpoint applies authentication methods, we still are vulnerable to attacks.

### PR DESCRIPTION
**Pull request by PedroCarrillo**
Fix #10 
Fix #11

### CODE REVIEW

File: app/src/main/java/com/belatrixsf/allstars/ui/account/AccountFragment.java [hunk 1]
```diff
@@ -118,6 +123,20 @@ public void showLevel(String level) {
...
 
+    @Override
+    public void showRole(String role) {
+        roleTextView.setText(role);
+    }
 
+    @Override
+    public void showProfilePicture(final String profilePicture) {
+        int width = getActivity().getResources().getDimensionPixelSize(R.dimen.dimen_10_8);
+        int height = getActivity().getResources().getDimensionPixelSize(R.dimen.dimen_10_8);
```
Comment (gyosida) [inline app/src/main/java/com/belatrixsf/allstars/ui/account/AccountFragment.java]: both variables have the same value.

File: app/src/main/java/com/belatrixsf/allstars/adapters/MainNavigationViewPagerAdapter.java [hunk 2]
```diff
@@ -25,6 +25,8 @@
 import android.support.v13.app.FragmentPagerAdapter;
 
 import com.belatrixsf.allstars.ui.account.AccountFragment;
+import com.belatrixsf.allstars.ui.example.ExampleFragment;
+import com.belatrixsf.allstars.ui.login.LoginFragment;
```
Comment (gyosida) [inline app/src/main/java/com/belatrixsf/allstars/adapters/MainNavigationViewPagerAdapter.java]: unused imports?

File: app/src/main/res/layout/fragment_account.xml [hunk 3]
```diff
@@ -108,30 +146,32 @@
...
-        </android.support.v7.widget.CardView>
-    </LinearLayout>
+                android:layout_height="match_parent"/>
+
+        </LinearLayout>
+
+        <ImageView
+            android:id="@+id/profile_picture"
+            android:layout_width="@dimen/dimen_10_8"
+            android:layout_height="80dp"
```
Comment (gyosida) [inline app/src/main/res/layout/fragment_account.xml]: should use @dimen/dimen_10_8

File: app/src/main/res/layout/fragment_account.xml [hunk 4]
```diff
@@ -6,27 +6,66 @@
...
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:layout_marginTop="@dimen/dimen_1_5"
+                        tools:text="Senior Android Developer" />
+
+                </LinearLayout>
+
+            </LinearLayout>
+
+            <View
```
Comment (gyosida) [inline app/src/main/res/layout/fragment_account.xml]: can you create a style for dividers?

Comment (PedroCarrillo) [inline app/src/main/res/layout/fragment_account.xml]: no

**Pull request merged**